hoomd/hpmc/pytest/test_shape.py

```diff
@@ -19,6 +19,10 @@ def check_dict(shape_dict, args):
     Useful for more complex nested dictionaries (like the shape key in unions)
     Used to test that the dictionary passed in is what gets passed out
     """
+    try:
+        shape_dict = shape_dict.to_base()
+    except AttributeError:
+        pass
```

Comment:
What raises an `AttributeError`? I haven't looked at the test data, but perhaps there is some kind of check you can have here to make sure that those cases are reasonable and that this won't ignore real problems?

Reply:
I need to examine this code again to remember why it is okay and necessary.